src/js/components/DataTable/index.d.ts

```diff
@@ -142,6 +142,7 @@ export interface DataTableProps<TRowType = any> {
         body?: VerticalAlignType;
         footer?: VerticalAlignType;
       };
+  allowSelectAll?: boolean;
```

Comment:
Super minor comment but can we alphabetize this prop (it should appear under `alignSelf` and above `background`)